lib/mutant/mutator/node/send.rb

```diff
@@ -223,7 +223,7 @@ def emit_argument_propagation
 
           argument = Mutant::Util.one(arguments)
 
-          return if n_kwargs?(argument) || n_forwarded_args?(argument)
+          return if n_kwargs?(argument) || n_forwarded_args?(argument) || n_forwarded_restarg?(argument)
```

Comment:
I wonder if this should check for forwarded keyword arguments too? https://github.com/mbj/mutant/pull/1401

Reply:
no as this is covered by `n_kwargs?` as `kwrestarg` forwarding is inside `s(:kwargs)` nodes.